index.html

```diff
@@ -9,16 +9,26 @@
 </head>
 <body>
     <main id="root">
-    <header>
-        <nav class="navbar">
+        <header class="navbar">
             <div id="logo">
-                <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
-            </div>
+                 <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
+             </div>
+             <nav>
+                   <div id="img">
+                       <img class="hamburger" src="./images/Union.png" alt="">
+                   </div>           
+               </nav>
+           </header>
+    <!-- <header>
+     <div id="logo">
+          <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
+      </div>
+      <nav class="navbar">
```

Comment:
- [ ] Great job making the changes suggested, however please notice that as mentioned in the best practices guidelines , we **Do not commit old pieces of code as inline comments. They will make your project look messy.** because of this I cannot approve the project yet, please fix this :+1: